moodle/Sniffs/Commenting/FileHasSniff.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
Isn't it name a little bit strange? Maybe 'FileExpectedTagsSniff' or 'FileHasTags' or ...

(not critical stuff in any case, for your consideration 100%).

Reply:
I like `FileExpectedTagsSniff`. Stolen :)